bigchaindb/backend/localmongodb/schema.py

```diff
@@ -32,118 +64,22 @@ def create_database(conn, dbname):
 @register_schema(LocalMongoDBConnection)
 def create_tables(conn, dbname):
     for table_name in backend.schema.TABLES:
-        logger.info('Create `%s` table.', table_name)
         # create the table
         # TODO: read and write concerns can be declared here
-        conn.conn[dbname].create_collection(table_name)
+        try:
+            logger.info(f'Create `{table_name}` table.')
+            conn.conn[dbname].create_collection(table_name)
+            create_indexes(conn, dbname, table_name, INDEXES[table_name])
+        except CollectionInvalid:
```

Comment:
The class name of this sample seems generic, is there a way to specifically determine that the error is in fact because of the fact that the collection already exists?

Reply:
I did not manage to find anything more specific. `pymongo` docs explain:

> CollectionInvalid will be raised if the collection already exists.